browser_patches/firefox/juggler/content/FrameTree.js

```diff
@@ -102,12 +102,25 @@ class FrameTree {
   }
 
   _onDOMWindowCreated(window) {
+    if (!window.__plawrightScrollbarsStylesheet && this.scrollbarsHidden) {
```

Comment:
Why the flag? Do we come here multiple times? If so, let's maybe use a symbol instead?

Reply:
Yeah, there is an event `juggler-dom-window-reused`.